lib/seahorn/BvOpSem2FatMemMgr.cc

```diff
@@ -73,76 +101,92 @@ class FatMemManager : public OpSemMemManager {
   ~FatMemManager() override = default;
 
   FatPtrTy ptrSort() const override {
-    return sort::structTy(m_mem.ptrSort(), m_ctx.alu().intTy(g_slotBitWidth),
+    return sort::structTy(m_main.ptrSort(), m_ctx.alu().intTy(g_slotBitWidth),
```

Comment:
the bitwidth of the slot has to correspond to the word width of the corresponding memory manager.

That is, `g_slotBitWith` and the word width of the slot memory manager must be the same size.

Reply:
Ack